src/lib/cozy-client/slices/sharings.js

```diff
@@ -425,7 +425,7 @@ const getSharingRecipients = (state, sharings) =>
     .reduce((a, b) => a.concat(b), [])
 
 const buildSharingLink = (id, doctype, sharecode) =>
-  `${window.location.origin}/public?sharecode=${sharecode}&id=${id}${
+  `/public?sharecode=${sharecode}&id=${id}${
```

Comment:
Would it be possible tu return the full link here ? It seems weird that `buildSharingLink ` returns a relative link, but I don't know what's possible.

Reply:
In this function, which is called by a selector, we have no access to the client instance and though to the cozy url. I can't see a quick, better way of doing.